codalab/worker/bundle_state.py

```diff
@@ -218,6 +220,7 @@ def from_dict(cls, dct):
             remote=dct['remote'],
             exitcode=dct['exitcode'],
             failure_message=dct['failure_message'],
+            bundle_profile_stats=dct['bundle_profile_stats'],
```

Comment:
Let's do this:
```suggestion
            bundle_profile_stats=dct.get('bundle_profile_stats'),
```

because that way existing bundles that are running during deployment (which won't have the bundle_profile_stats key set) will be backwards compatible

Reply:
Should we make that the case for every field? for the sake of consistency

Reply:
Sure, good point -- feel free to